db/migrate/20230616212030_add_user_to_likes.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserToLikes < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, foreign_key: true
+  end
+end
```

Comment:
- [ ] Kindly remove this migration as it's throwing errors. you are trying to add the user reference in like but this was already added.
<img width="779" alt="image" src="https://github.com/Itswali/blog_app/assets/79058364/ce0710c0-7fed-4cfd-a43c-3f0c4f4c73a8">